parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -312,6 +312,11 @@ ensureKubelet() {
         sleep 3
     done
     {{end}}
+    {{if HasAntreaNetworkPolicy}}
+    while [ ! -f /etc/cni/net.d/10-antrea.conf ]; do
```

Comment:
Note: if the daemonset implementation ever changes the name lf this CNI config file, we'll have to update this file wait implementation.